### PR TITLE
Initial instructions for Amazon Linux 2

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

**Comment by amarantha-k** [comment]
@mihirkamdarcouchbase Could you please review the instructions?

**Comment by amarantha-k** [comment]
Got +1 from Mihir over Slack. 
@eric-schneider Could you review the PR please? Need an approval to merge the updates.

File: modules/install/pages/amazon-linux2-install.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+= Amazon Linux 2 Installation
+:page-edition: enterprise
+
+[abstract]
+Couchbase Server version 6.0.1 adds support for the Amazon Linux 2 platform.
```
Comment (eric-schneider) [inline modules/install/pages/amazon-linux2-install.adoc]:
```suggestion
Instructions for installing Couchbase Server on Amazon Linux 2.
```

File: modules/install/pages/amazon-linux2-install.adoc [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
```
Comment (eric-schneider) [inline modules/install/pages/amazon-linux2-install.adoc:6]:
```suggestion

Amazon Linux 2 is supported with Couchbase Server 6.0.1+.
See xref:install-platforms.adoc[Supported Operating Systems] for details.

```

File: modules/install/pages/amazon-linux2-install.adoc [hunk 3]
```diff
@@ -0,0 +1,114 @@
+= Amazon Linux 2 Installation
+:page-edition: enterprise
+
+[abstract]
+Couchbase Server version 6.0.1 adds support for the Amazon Linux 2 platform.
```
Comment (eric-schneider) [inline modules/install/pages/amazon-linux2-install.adoc]:
```suggestion
Instructions for installing Couchbase Server on Amazon Linux 2.
```

File: modules/install/pages/amazon-linux2-install.adoc [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
```
Comment (eric-schneider) [inline modules/install/pages/amazon-linux2-install.adoc:9]:
```suggestion

Amazon Linux 2 is supported with Couchbase Server 6.0.1+.
See xref:install-platforms.adoc[Supported Operating Systems] for details.

```

**Review by eric-schneider — CHANGES_REQUESTED** [review]

**Comment by amarantha-k** [comment]
@eric-schneider Changes incorporated. Could you take a look and approve?

**Review by eric-schneider — APPROVED** [review]

**Pull request merged**